### PR TITLE
Feature/jjdd8 fpo 6 fix parser

### PR DESCRIPTION
**Pull request by DariuszCzajkowski**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/BookPrinter.java [hunk 1]
```diff
@@ -30,91 +31,84 @@ public void printBooks(List<Book> books) {
...
                     }
                     counter = 0;
-                    ClearScreen.clearScreen();
-                }
+                    ScreenCleaner.clearScreen();
             }
         }
-        menuBookList();
+        if (!isExit) {
+            menuBookList();
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/BookPrinter.java]: metoda powinna mieć formę czasownikową

**Pull request merged**